src/components/FlatList/index.tsx

```diff
@@ -54,7 +54,6 @@ function MVCPFlatList<TItem>({maintainVisibleContentPosition, horizontal = false
         return horizontal ? getScrollableNode(scrollRef.current)?.scrollLeft ?? 0 : getScrollableNode(scrollRef.current)?.scrollTop ?? 0;
     }, [horizontal]);
 
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
How is it connected to this PR?

Reply:
It was used here without a reason, so I removed it in here